travis/build_wheels.sh

```diff
@@ -47,60 +39,35 @@ cd -
 
 # Compile wheels
 for PYBIN in /opt/python/cp3*/bin; do
-    # Don't build for python 3.8 while tensorflow doesn't create wheels for it
-    # https://github.com/tensorflow/addons/issues/744
-    if [[ $WITH_TENSORFLOW ]] && [[ $PYBIN == *"cp38"* ]]; then break; fi
-    if [[ $WITH_TENSORFLOW ]] && [[ $PYBIN == *"cp39"* ]]; then break; fi
-
-    if [[ $WITH_TENSORFLOW ]]; then
-    # The minimum numpy version required by tensorflow is always greater than
-    # the installed one. Install the oldest numpy supported by each tensorflow
-    # to get the maximum fordwards compatibility
-        NUMPY_VERSION=$( "${PYBIN}/pip" check tensorflow |grep tensorflow |grep numpy |grep ">=" |awk -F"[>=',]+" '//{print $2}' )
-
-        if [[ ${NUMPY_VERSION} ]]; then
-            echo "Got numpy ${NUMPY_VERSION} from the Tensorflow requirements"
-            "${PYBIN}/pip" install numpy==$NUMPY_VERSION
-        fi
-
-        "${PYBIN}/pip" install tensorflow==$TENSORFLOW_VERSION
-
-        # Make the tensorflow symbolic links point to the shared libraries
-        # installed with the tensorflow wheel
-        "${PYBIN}/python" /io/src/3rdparty/tensorflow/setup_tensorflow.py -m python -c "${PREFIX}"
-
-    else
     # Use the oldest version of numpy for each Python version
     # for backwards compatibility of its C API
     # https://github.com/numpy/numpy/issues/5888
     # Build numpy versions used by scikit-learn:
     # https://github.com/MacPython/scikit-learn-wheels/blob/master/.travis.yml
 
-        # Python 2.7
-        NUMPY_VERSION=1.8.2
-
-        # Python 3.x
-        if [[ $PYBIN == *"cp39"* ]]; then
-            NUMPY_VERSION=1.19.3
-        elif [[ $PYBIN == *"cp38"* ]]; then
-            NUMPY_VERSION=1.17.4
-        elif [[ $PYBIN == *"cp37"* ]]; then
-            NUMPY_VERSION=1.14.5
-        elif [[ $PYBIN == *"cp36"* ]]; then
-            NUMPY_VERSION=1.11.3
-        elif [[ $PYBIN == *"cp34"* ]] || [[ $PYBIN == *"cp35"* ]]; then
-            NUMPY_VERSION=1.9.3
-        fi
-
-        "${PYBIN}/pip" install numpy==$NUMPY_VERSION
+    # Python 2.7
+    NUMPY_VERSION=1.8.2
+
+    # Python 3.x
+    if [[ $PYBIN == *"cp39"* ]]; then
+        NUMPY_VERSION=1.19.3
+    elif [[ $PYBIN == *"cp38"* ]]; then
+        NUMPY_VERSION=1.17.4
+    elif [[ $PYBIN == *"cp37"* ]]; then
+        NUMPY_VERSION=1.14.5
+    elif [[ $PYBIN == *"cp36"* ]]; then
+        NUMPY_VERSION=1.11.3
+    elif [[ $PYBIN == *"cp34"* ]] || [[ $PYBIN == *"cp35"* ]]; then
```

Comment:
We discard these outdated Python versions, so should this be commented? It may be good to keep the required NUMPY_VERSION for them documented.

Reply:
Ok, I will comment  out code related to outdated Python versions.